File: WebApplication/Models/game.cs
namespace WebApplication
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("game")]
    public partial class game
    {
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2214:DoNotCallOverridableMethodsInConstructors")]
        public game()
        {
            game_moves = new HashSet<game_moves>();
        }

        public int id { get; set; }

        public int whiteUserId { get; set; }

        public int blackUserId { get; set; }

        [Column(TypeName = "datetime2")]
        public DateTime startDate { get; set; }

        [Column(TypeName = "datetime2")]
        public DateTime? endDate { get; set; }

        public short? winner { get; set; }

        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
        public virtual ICollection<game_moves> game_moves { get; set; }

        public virtual users users { get; set; }

        public virtual users users1 { get; set; }
    }
}

File: WebPage/game.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.Data.Entity.Spatial

<Table("game")>
Partial Public Class game
    Public Sub New()
        game_moves = New HashSet(Of game_moves)()
    End Sub

    Public Property id As Integer

    Public Property whiteUserId As Integer

    Public Property blackUserId As Integer

    <Column(TypeName:="datetime2")>
    Public Property startDate As Date

    <Column(TypeName:="datetime2")>
    Public Property endDate As Date?

    Public Property winner As Short?

    Public Overridable Property game_moves As ICollection(Of game_moves)

    Public Overridable Property users As users

    Public Overridable Property users1 As users
End Class
